news/2017-11-29-mapping-with-rewards-2-bounties-complete.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post 
+title: "Mapping With Rewards #2 - Bounties Complete!"
+date: 2017-11-29
```

Comment:
`date: 2017-12-01 hh:mm:ss: +0000`
time of post missing

Reply:
delayed